src/pages/ComponentsPage/ComponentsPage.js

```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
 import Loader from '../../components/Loader/Loader';
 import './ComponentsPage.sass';
 import COMPONENTS from '../../app/components';
+import Sidebar from '../../components/Sidebar/Sidebar';
+import Searchbar from '../../components/Searchbar/Searchbar';
+import ComponentDocsList from '../../components/ComponentDocsList/ComponentDocsList';
```

Comment:
Completely off topic, but we should revisit the naming convention for the main component files. The OCD programmer in me is starting to freak out over the redundancy of `../../components/MyIncredibleThing/MyIncredibleThing`